geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/CommandIntegrationTest.java

```diff
@@ -81,6 +87,32 @@ public void commandReturnsResultsMatchingNativeRedis() {
     softly.assertAll();
   }
 
+  @Test
+  public void commandWithInvalidSubcommand_returnError() {
+    String invalidSubcommand = "fakeSubcommand";
+    RedisCodec<String, String> codec = StringCodec.UTF8;
+
+    CommandArgs<String, String> args =
+        new CommandArgs<>(codec).add(CommandType.COMMAND).add(invalidSubcommand);
+    assertThatThrownBy(
+        () -> radishClient.dispatch(CommandType.COMMAND, new NestedMultiOutput<>(codec), args))
+            .hasMessageContaining(
+                "ERR Unknown subcommand or wrong number of arguments for 'COMMAND'");
```

Comment:
it would be good to use `ERROR_UNKNOWN_COMMAND_COMMAND_SUBCOMMAND` from `RedisConstants` so that if we change the error message it'll still work. I don't think we're really testing that the constant is right, just that the constant we expect gets returned.